WordPress/WordPress-Internal-Info.plist

```diff
@@ -464,6 +464,7 @@
 	<false/>
 	<key>LSApplicationQueriesSchemes</key>
 	<array>
+		<string>jetpacknotificationmigration</string>
```

Comment:
We might need to declare the URL scheme that WP owns in this and any other plists.